Mathlib/Algebra/Equiv/TransferInstance.lean

```diff
@@ -1,372 +1,5 @@
-  intros
-  letI : Module R α := e.module R
-  fapply Algebra.ofModule
-  · intro r x y
-    change e.symm (e (e.symm (r • e x)) * e y) = e.symm (r • e.ringEquiv (x * y))
-    simp only [apply_symm_apply, Algebra.smul_mul_assoc, map_mul, ringEquiv_apply]
-  · intro r x y
-    change e.symm (e x * e (e.symm (r • e y))) = e.symm (r • e (e.symm (e x * e y)))
-    simp only [apply_symm_apply, Algebra.mul_smul_comm]
-
-lemma algebraMap_def (e : α ≃ β) [Semiring β] [Algebra R β] (r : R) :
-    (@algebraMap R α _ (Equiv.semiring e) (Equiv.algebra R e)) r = e.symm ((algebraMap R β) r) := by
-  let _ := Equiv.semiring e
-  let _ := Equiv.algebra R e
-  simp only [Algebra.algebraMap_eq_smul_one]
-  change e.symm (r • e 1) = e.symm (r • 1)
-  simp only [Equiv.one_def, apply_symm_apply]
-
-/-- An equivalence `e : α ≃ β` gives an algebra equivalence `α ≃ₐ[R] β`
-where the `R`-algebra structure on `α` is
-the one obtained by transporting an `R`-algebra structure on `β` back along `e`.
--/
-def algEquiv (e : α ≃ β) [Semiring β] [Algebra R β] : by
-    let semiring := Equiv.semiring e
-    let algebra := Equiv.algebra R e
-    exact α ≃ₐ[R] β := by
-  intros
-  exact
-    { Equiv.ringEquiv e with
-      commutes' := fun r => by
-        apply e.symm.injective
-        simp only [RingEquiv.toEquiv_eq_coe, toFun_as_coe, EquivLike.coe_coe, ringEquiv_apply,
-          symm_apply_apply, algebraMap_def] }
-
-@[simp]
-theorem algEquiv_apply (e : α ≃ β) [Semiring β] [Algebra R β] (a : α) : (algEquiv R e) a = e a :=
-  rfl
-
-theorem algEquiv_symm_apply (e : α ≃ β) [Semiring β] [Algebra R β] (b : β) : by
-    letI := Equiv.semiring e
-    letI := Equiv.algebra R e
-    exact (algEquiv R e).symm b = e.symm b := rfl
-
-end
-
-end R
-
-end Instances
-
-end Equiv
-
-section
-
-variable {R : Type*} [CommSemiring R]
-variable (A : Type*) [Semiring A] [Algebra R A]
-variable [AddCommMonoid α] [AddCommMonoid β] [Module A β]
-
-/-- Transport a module instance via an isomorphism of the underlying abelian groups.
-This has better definitional properties than `Equiv.module` since here
-the abelian group structure remains unmodified. -/
-abbrev AddEquiv.module (e : α ≃+ β) :
-    Module A α where
-  toSMul := e.toEquiv.smul A
-  one_smul := by simp [Equiv.smul_def]
-  mul_smul := by simp [Equiv.smul_def, mul_smul]
-  smul_zero := by simp [Equiv.smul_def]
-  smul_add := by simp [Equiv.smul_def]
-  add_smul := by simp [Equiv.smul_def, add_smul]
-  zero_smul := by simp [Equiv.smul_def]
-
-/-- The module instance from `AddEquiv.module` is compatible with the `R`-module structures,
-if the `AddEquiv` is induced by an `R`-module isomorphism. -/
-lemma LinearEquiv.isScalarTower [Module R α] [Module R β] [IsScalarTower R A β]
-    (e : α ≃ₗ[R] β) :
-    letI := e.toAddEquiv.module A
-    IsScalarTower R A α := by
-  letI := e.toAddEquiv.module A
-  constructor
-  intro x y z
-  simp only [Equiv.smul_def, smul_assoc]
-  apply e.symm.map_smul
-
-end
+deprecated_module (since := "2025-07-04")
```

Comment:
I think we decided that these don't belong in PRs, and should be generated by a bot in a later PR instead.

Reply:
Sure, but I thought that in this case there was nothing to be gained by waiting since no single file is getting most of the content of the existing one